### PR TITLE
Remove user-specific files from VCS

### PR DESCRIPTION
**Pull request by redbassett**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -60,3 +60,10 @@ freeline_project_description.json
 .idea/
 app/src/main/gen/
 app/src/main/main.iml
+
+# Ignore Android Studio .iml files
+*.iml
+
```
Comment (kimj) [inline .gitignore]: isn't *.iml already in this (under #IntelliJ). you may be running into an issue where sometime files have to be specified.

File: .gitignore [hunk 2]
```diff
@@ -60,3 +60,10 @@ freeline_project_description.json
...
+# Ignore local config properties
+local.properties
+
```
Comment (kimj) [inline .gitignore]: same here

**Comment by emezias** [comment]
@redbassett I used the Studio instructions to "handle ignored and tracked files". That created a merge conflict. See if there's anything left from here to fix in the project.

**Comment by redbassett** [comment]
@kimj was correct about duplicate lines.

Removed `.gitignore` from `.gitignore`. Merges fine on my local machine, removing cached versions of `app/app.iml` and `local.properties`.

**Comment by redbassett** [comment]
New PR resolving conflicts: https://github.com/emezias/AndroidCVCodeLab/pull/5

**Pull request closed without merging**